src/Microsoft.Azure.WebJobs/IBinderExtensions.cs

```diff
@@ -5,8 +5,10 @@
 
 namespace Microsoft.Azure.WebJobs
 {
-    /// <summary>Provides extension methods for the <see cref="IBinder"/> interface.</summary>
-    public static class BinderExtensions
+    /// <summary>
+    /// Provides extension methods for the <see cref="IBinder"/> interface.
+    /// </summary>
+    public static class IBinderExtensions
```

Comment:
Renaming this class to avoid type name conflicts with the new extension class I'm adding. Should have been named this to begin with.

Since people don't reference this extension class directly by type name, this technically breaking change should be fine.